examples/admin-client/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <artifactId>keycloak-examples-parent</artifactId>
         <groupId>org.keycloak</groupId>
-        <version>8.0.0-SNAPSHOT</version>
+        <version>9.0.0-SNAPSHOT</version>
```

Comment:
Could you remove the version change here? There will be probably some other (different) taks to cleanup this. This applies to all pom.xml files.